sbi/inference/snl/snl.py

```diff
@@ -251,7 +251,7 @@ def _fit_likelihood(
         # Get total number of training examples.
         num_examples = torch.cat(self._parameter_bank).shape[0]
 
-        # Select random train and validation splits from (parameter, observation) pairs.
+        # Select random neural_net and validation splits from (parameter, observation) pairs.
```

Comment:
Why this change? Is this a search and replace mistake from `train` to `neural_net`?

Reply:
same for `sbi/inference/sre.py`